docs/options.md

```diff
@@ -255,7 +255,7 @@ Default: `auto`
 
 - For `linux`, you need [Docker or Podman](#container-engine) running, on Linux, macOS, or Windows.
 - For `macos` and `windows`, you need to be running on the respective system, with a working compiler toolchain installed - Xcode Command Line tools for macOS, and MSVC for Windows.
-- For `pyodide` you need to be on an x86-64 linux runner and `python3.12` must be available in `PATH`.
+- For `pyodide` `python3.12` must be available in `PATH` and you need to be on one of the following runners: x86-64 Linux, arm64 Linux. Intel and Silicon macOS hosts may work, too. See [the section on Pyodide](setup.md#pyodide-(WebAssembly)-builds-(experimental)) for more information.
```

Comment:
Why don't we keep telling people to use x86-64 linux runners? Is there a good reason to use arm64 linux? We don't test it ourselves, are we sure that it works? "Mac builds may succeed but there are known bugs".

Reply:
Added your phrasing here: 057e542fa3a670dd4e92773e0e33a12b806dbc6a. I'd like to see if there is interest in adding Linux-arm64 and macOS-arm64 builds, so I've kept this conversation as "unresolved" for now :)